src/json-utils.js

```diff
@@ -175,7 +178,7 @@ const renameProps = {
   intersection: 'not-intersection'
 };
 
-function filterJSONstreet (removeProps, renameProps, streetJSON) {
```

Comment:
is this function called from 3dstreet editor? it may cause this error
https://github.com/3DStreet/3dstreet-editor/blob/f763a949ce4cd643573ac7f3f7e321d8bbfa8c6a/src/components/scenegraph/Toolbar.js#L130C26-L130C37

Reply:
Yes, that is why this PR should be applied to editor after to prevent that error:
https://github.com/3DStreet/3dstreet-editor/pull/387